sites/public/src/pages/sign-in.tsx

```diff
@@ -72,29 +72,38 @@ const SignIn = (props: SignInProps) => {
     isLoading: isResendConfirmationLoading,
   } = useMutate<SuccessDTO>()
 
-  const setLocalStorage = () => {
-    window.localStorage.setItem(
-      "bloom-show-favorites-menu-item",
-      (
-        isFeatureFlagOn(props.jurisdiction, FeatureFlagEnum.enableListingFavoriting) === true
-      ).toString()
-    )
-  }
-
   useEffect(() => {
     pushGtmEvent<PageView>({
       event: "pageView",
       pageTitle: "Sign In",
       status: UserStatus.NotLoggedIn,
     })
     if (props.jurisdiction) {
-      setLocalStorage()
+      setFeatureFlagLocalStorage(
+        props.jurisdiction,
+        FeatureFlagEnum.enableListingFavoriting,
+        "show-favorites-menu-item"
+      )
+      setFeatureFlagLocalStorage(
+        props.jurisdiction,
+        FeatureFlagEnum.disableCommonApplication,
+        "hide-applications-menu-item"
+      )
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
   useEffect(() => {
-    setLocalStorage()
+    setFeatureFlagLocalStorage(
```

Comment:
I will admit, I don't understand why the same code needs to run separately in two different effect hooks, but I kept the logic from previously since it sounds like it fixed a bug of some kind.